File: Resource/51900286-51900712-51900718-51900699/source demo/Web Application/ComNha_API/ComNha_API/Models/Chitietgoimon.cs
using System;
using System.Collections.Generic;

namespace ComNha_API.Models
{
    public partial class Chitietgoimon
    {
        public string MaMonAn { get; set; } = null!;
        public string MaOrder { get; set; } = null!;
        public int SoLuongMonAn { get; set; }

        public virtual Monan MaMonAnNavigation { get; set; } = null!;
        public virtual Phieugoimon MaOrderNavigation { get; set; } = null!;
    }
}

File: References/restaurent-management-comnha/src/main/java/tdtu/thinh6/restaurent_management_comnha/domain/Chitietgoimon.java
package tdtu.thinh6.restaurent_management_comnha.domain;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import lombok.Getter;
import lombok.Setter;


@Entity
@Getter
@Setter
public class Chitietgoimon {

    @Id
    @Column(nullable = false, updatable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer soLuongMonAn;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ma_mon_an_id", nullable = false)
    private Monan maMonAn;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ma_order_id", nullable = false)
    private Phieugoimon maOrder;

}
